advocacy_docs/partner_docs/CohesityDataProtectforPostgreSQL/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+Using Cohesity DataProtect for PostgreSQL, you can backup and restore your EDB Postgres Advanced Server and PostgreSQL databases, set specific backup policies for them, and keep track of the backup and restore runs. Cohesity DataProtect for PostgreSQL provides peace of mind for database owners with its nontraditional approaches to backups.  The Cohesity platform natively integrates with PostgreSQL to provide backup and recovery for PostgreSQL deployments. Cohesity’s Connector Agents allow for flexibility of customization, including being part of Cohesity’s data management platform, Helios.
+
+The Cohesity platform provides a PostgreSQL connector that enables you to perform full backups, incremental (differential) backups, log backups, restores, and point-in-time restores, all at the cluster level.
+
+The following are the core components for protecting your EDB Postgres Advanced Server or PostgreSQL database with Cohesity DataProtect for PostgreSQL:
+
+- EDB Postgres Advanced Server or PostgreSQL database
+- Cohesity Backup Appliance
+- Cohesity PostgreSQL Connector
+- Cohesity Connector Agent
+
+
+The diagram below shows how you can integrate the Cohesity platform with your EDB Postgres Advanced Server and PostgreSQL deployments using Cohesity DataProtect for PostgreSQL.
```

Comment:
Diagram is not showing how you can integrate.
I think we can remove the entire sentence.  See Veritas solution summary as an example.